web_src/css/repo/issue-list.css

```diff
@@ -82,6 +82,10 @@
   background-color: var(--color-secondary-dark-4);
 }
 
+#issue-list .separator-container > .flex-text-inline::before {
+  content: "•";
```

Comment:
I guess it could avoid such CSS style, just output a `•` in HTML?

Reply:
Its quite difficult because of the large amount of 'if'-statement there needs to be a check for each case, that there is (no) content after it and show depending of the content